sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_shared/challenge_auth_policy.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, credential, **kwargs):
 
     def send(self, request):
         # type: (PipelineRequest) -> HttpResponse
+        self._enforce_tls(request)
```

Comment:
I didn't see this defined in the PR, but assuming it already was.

Reply:
It's inherited from a base class shared with the more general bearer token policy.